extensions-contrib/azure-extensions/src/main/java/org/apache/druid/firehose/azure/AzureBlob.java

```diff
@@ -26,6 +26,7 @@
 import java.util.Objects;
 
 
+@Deprecated
```

Comment:
Add instructions about how to move away from this deprecated class and if there are any benefits, why the new way is preferred.

Similar comments for other deprecated annotations